source/Boardwalk-Tests/IdentifierAssignerTest.class.st

```diff
@@ -16,14 +17,16 @@ IdentifierAssignerTest >> id: anId [
 { #category : #'test support' }
 IdentifierAssignerTest >> nextId [
 
-	^ '123'
+	nextId := nextId + 1.
```

Comment:
Why are you doing this as an arithmetic operation instead of writing it like this 

```
  | currentId |
  currentId := nextId.
  nextId := nextId + 1.
  ^ currentId
```